tests/fixtures/config/env/entity.yml

```diff
@@ -1,5 +1,5 @@
 # this is a comment at the top of the file
-include: ./empty_config.yml
+include: ./includes/empty_config.yml
```

Comment:
Projects being setup by setup_project do not like empty environment files right under `env`. Moving them to the includes folder makes the purpose clear and also fixes the bug.